src/MasterSite.php

```diff
@@ -195,6 +195,15 @@ function ( $block_content, $block ) {
 		add_action( 'admin_notices', [ $this, 'show_dashboard_notice' ] );
 		add_action( 'wp_ajax_dismiss_dashboard_notice', [ $this, 'dismiss_dashboard_notice' ] );
 		add_filter( 'timber/twig', [ $this, 'p4_optimize_img_url' ] );
+
+		// Disable WordPress(WP5.5) Block Directory.
+		add_action(
+			'plugins_loaded',
+			function() {
+				remove_action( 'enqueue_block_editor_assets', 'wp_enqueue_editor_block_directory_assets' );
+				remove_action( 'enqueue_block_editor_assets', 'gutenberg_enqueue_block_editor_assets_block_directory' );
+			}
+		);
```

Comment:
OK, I can't make the block directory work for some reason (it makes an OPTION request to its endpoint when I type in the box and that's all, no search after that); but I can see the script `block-directory.js` still loaded in the network tab.
From what I've read and tested, you don't need a hook to [remove an action from a theme](https://github.com/WordPress/gutenberg/issues/23961#issuecomment-688957626), and I can't see where the `gutenberg_enqueue_block_editor_assets_block_directory` is used, I can only assume it is specific to the Gutenberg repo and transformed at some point, but not included in the Wordpress install. In the end using only this one (without `add_action()`) seems to work:
```php
remove_action( 'enqueue_block_editor_assets', 'wp_enqueue_editor_block_directory_assets' );
```
Which version are you on ? I just installed 5.5.3 so maybe we are on a different one (5.5.2 was just a few days before)

Reply:
Hi @lithrel - Thanks for the review :) . I am using the same WPv5.5.3 version. Could you try commenting the `define('DISALLOW_FILE_MODS', true );` line in `wp-config.php` file.

I updated the PR with above changes. please have a look.

Reply:
> Could you try commenting the `define('DISALLOW_FILE_MODS', true );` line in `wp-config.php` file.

That was it, thanks !